.FSC-PS/environments.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "name":"Dummy",
+    "settings":{
+      "buildVersion": "10.0.38",
```

Comment:
Let's see if this affects the build. The action in the source branch seems to use 10.0.40, so all should be good